### ISSUE TITLE
omitChecksum -> computeChecksum

### ISSUE THREAD

**Issue by nschloe** [timdown/rangy#284]
The rangy functions `serialize*` have an optional argument `omitChecksum`, e.g.,
```
rangy.serializeSelection([Selection sel[, Boolean omitChecksum, [Element rootNode]])
```
If `true`, the checksum is _not_ computed.

Negating argument names is most often not a good idea because this makes it harder to understand code. It takes a while to realize that
```
rangy.serializeSelection(selection, false);
```
actually _does_ compute the checksum since `false` hints that something does _not_ happen.

A better way to do this would be to replace the argument `omitChecksum` by, e.g., `computeChecksum`, and have it set to `true` by default (or whatever the current default is).

I realize of course that this change is breaking the API, so arguably you don't want to consider it before your next API-breaking release.

**Comment by timdown** [comment]
I agree. I think my reasoning at the time was that the current API would be the easiest to code, which is a terrible reason. I favour using an options object now, so a future release is likely to introduce an options object while retaining compatibility.
